src/components/client/core/Drawer/Drawer.spec.tsx

```diff
@@ -0,0 +1,37 @@
+import { expect } from "@storybook/jest";
+import { screen, within } from "@storybook/testing-library";
+
+import { sleep } from "lib/utils";
+
+import type { ReactRenderer } from "@storybook/react";
+import type { PlayFunctionContext, Renderer } from "@storybook/types";
+
+// TODO implement `userEvent` simulation package from `@storybook/testing-library` instead of native HTML browser click event (e.g. `await userEvent.click(openButton);` instead of `openButton.click();`); https://trello.com/c/Ez0nSBmA/152-implement-userevent-in-story-play-tests
+
+/**
+ * Test drawer opening and closing.
+ */
+export const openState = async <R extends Renderer = ReactRenderer>({
+  canvasElement,
+}: PlayFunctionContext<R>) => {
+  const canvas = within(canvasElement as HTMLElement);
+
+  const openButton = canvas.getByRole("button", {
+    name: /open drawer/i,
+  });
+
+  openButton.click();
```

Comment:
Could not get RTL `userEvent` wrapped in Storybook test package working here, which is better practice than native HTML click event (see TODO at top of file)